src/mainboard/aaeon/upsquared/fixed-dtfs.dts

```diff
@@ -6,9 +6,9 @@
 
     flash-info {
```

Comment:
I think you should delete this file -- it's upsquared while everything else is upxtreme.